task1/Program.cs

```diff
@@ -45,5 +56,175 @@ public static decimal PerformCalculation(int r)
             decimal square = Pi * r * r;
             return square;
         }
+
+        public static void HowManySymbolInText()
+        {
+            Console.WriteLine("Enter text");
+            string text = Console.ReadLine();
+            Console.WriteLine("What symbol do you want to count?");
+            char symbol = Convert.ToChar(Console.ReadLine());
+            int count = 0;
+            for (int i = 0; i < text.Length; i++)
+            {
+                if (text[i] == symbol)
+                    count++;
+            }
+            Console.WriteLine($"Symbol {symbol} is met {count} times");
+        }
+
+        public static void InsertString()
+        {
+            Console.WriteLine("Enter text");
+            string text = Console.ReadLine();
+            Console.WriteLine("Enter inserted text");
+            string insertedText = Console.ReadLine();
+            Console.WriteLine("In what position should text be inserted?");
+            int position = Convert.ToInt32(Console.ReadLine());
+            string result = text.Insert(position, insertedText);
+            Console.WriteLine(result);
+        }
+
+        public static void DeleteSpaces()
+        {
+            Console.WriteLine("Input text");
+            string text = Console.ReadLine();
+            //string result = string.Empty;
+
+            int index1 = text.IndexOf('?', 0);
+            int index2 = text.IndexOf('?', index1 + 1);
+            char[] result = new char[text.Length];
+            StringBuilder str = new StringBuilder();
+            str = str.Append(text);
+            str.Replace(" ", null, index1, index2 - index1);
```

Comment:
Please do not use null if it possible. In case with strings we should use string.Empty for all such cases.